package.json

```diff
@@ -7,6 +7,7 @@
     "chai": ">= 1.3.0",
     "js-yaml": ">= 1.0.3",
     "mocha": "^6.0.0",
+    "serialize-to-js": "^3.0.0",
```

Comment:
Specify the version as `>=3.0.0` or so. I don't want to update fixed versions next to next.

Reply:
But `>=` would allow for major versions higher than 3 and there could be breaking changes in those. It doesn't seem safe to me.

Reply:
(I really think that it should be safe because it's a library, but sometimes npm libraries are doing surprising things...)
Anyway, that dependency is only `devDependencies`, so it's ok.